utils/Constants.ts

```diff
@@ -48,28 +57,37 @@ export const SECONDARY_COLLATERAL_TOKEN_NAME: { [key: string]: string } = {
   rsk: "ETHs"
 };
 
-export const numberOfLPs = 11;
-
-export const initialBdstableMintingAmount = (networkName = "mainnetFork"): BigNumber => {
-  switch (networkName) {
-    case "rsk": {
-      return to_d18(15e3);
-    }
-    case "mainnetFork": {
-      return to_d18(30e3);
-    }
-    default: {
-      return to_d18(15e3);
-    }
-  }
-};
-
 export const INITIAL_BDEU_UNISWAP_EUR_AMOUNT = 500;
 export const INITIAL_BDUS_UNISWAP_USD_AMOUNT = 500;
+export const INITIAL_BXAU_UNISWAP_XAU_AMOUNT = 0.3;
+export const INITIAL_BGBP_UNISWAP_GBP_AMOUNT = 500;
 export const INITIAL_BDX_UNISWAP_EUR_AMOUNT = 9000;
 export const INITIAL_BDX_UNISWAP_USD_AMOUNT = 10000;
 export const INITIAL_BDX_AMOUNT_FOR_BDSTABLE = to_d18(1e5);
 
+const initalBDStablePerSymbol = {
+  rsk: {
+    XAU: to_d18(INITIAL_BXAU_UNISWAP_XAU_AMOUNT), // TODO: Define amount
+    GBP: to_d18(INITIAL_BGBP_UNISWAP_GBP_AMOUNT) // TODO: Define amount
```

Comment:
How much bGBP and bXAU do we want to mint on deployment and send over to the treasury?

Reply:
IMO almost nothing, $100 worth of each. Let's let the market do its thing. WDYT?

Reply:
sounds good